pages/attacks/Content_Spoofing.md

```diff
@@ -42,7 +42,7 @@ is properly done by the victim business. For public traded companies,
 its shares will be falling down, leading to uncontrolled loss of
 millions.
 
-![](../assets/images/Fake-text.png "Fake-text.png")
+![](assets/images/Fake-text.png "Fake-text.png")
```

Comment:
This one seems fine:
![image](https://user-images.githubusercontent.com/7570458/86347641-930e6b00-bc2c-11ea-9ab4-f2cc20063a98.png)